lib/router/router.dart

```diff
@@ -316,19 +316,36 @@ GoRouter createRouter(
               name: 'Webview',
               path: 'webview',
               parentNavigatorKey: rootNavigatorKey,
-              builder: (context, state) {
-                if (state.extra == null) {
-                  return const SizedBox();
-                }
+              pageBuilder: (context, state) {
 
                 final extra = state.extra as Map<String, dynamic>;
 
-                return WebViewScreen(
-                  url: extra['url'],
-                  redirectUrl: extra['redirectUrl'],
-                  customScheme: extra['customScheme'],
+                return CustomTransitionPage(
```

Comment:
Since we switched to using a modal, we don't need this route anymore, do we?

Reply:
in the send_details screen I now changed it to use the webview modal to show community topup screen